operator/utils/utils.go

```diff
@@ -18,6 +26,18 @@ func GetPredictiveUnitAsJson(params []machinelearningv1.Parameter) string {
 	}
 }
 
+func GetPredictionPath(mlDep *machinelearningv1.SeldonDeployment) string {
+	protocol := mlDep.Spec.Protocol
+
+	if protocol == "tensorflow" {
+		// Currently there is no easy way to create a reasonable access to tf models
+		// 	so for now we're making available only the first model
+		return "/v1/models/" + mlDep.Spec.Predictors[0].Graph.Name + "/:predict"
```

Comment:
I think we are thinking of having the SeldonDeployment name as the name of the model but actually needs further thought - maybe comment on https://github.com/SeldonIO/seldon-core/issues/1611

Reply:
Yeah I agree, happy to wait to see what gets confirmed. I like the idea of using "no model name as the default"

Reply:
I've updated comment so that it reflects the current discussion